mmv1/templates/terraform/examples/autoscaler_single_instance.tf.erb

```diff
@@ -1,5 +1,5 @@
 resource "google_compute_autoscaler" "<%= ctx[:primary_resource_id] %>" {
-  provider = google-beta
+  provider = google-beta.us-central1
```

Comment:
Would you mind explaining a bit about the change? I wonder if this will be considered as a breaking change.